File: app/src/main/java/com/solt9029/editmasterandroid/di/module/ActivityBuilderModule.java
package com.solt9029.editmasterandroid.di.module;

import com.solt9029.editmasterandroid.view.activity.ScoreActivity;
import com.solt9029.editmasterandroid.view.activity.ScoreListActivity;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class ActivityBuilderModule {
    @ContributesAndroidInjector
    abstract ScoreListActivity contributeScoreListActivity();

    @ContributesAndroidInjector
    abstract ScoreActivity contributeScoreActivity();
}

File: app/src/main/java/com/solt9029/editmasterandroid/di/module/ActivityBuilderModule.kt
package com.solt9029.editmasterandroid.di.module

import com.solt9029.editmasterandroid.view.activity.ScoreActivity
import com.solt9029.editmasterandroid.view.activity.ScoreListActivity
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module interface ActivityBuilderModule {
    @ContributesAndroidInjector
    fun contributeScoreListActivity(): ScoreListActivity

    @ContributesAndroidInjector
    fun contributeScoreActivity(): ScoreActivity
}
